packages/core/src/test/easing.test.ts

```diff
@@ -0,0 +1,52 @@
+import {Easing} from '../easing';
+
+const numbersToTest = [0, 0.2, 0.5, 0.7, 1];
+
+test('Easing linear', () => {
+	const easingIn = Easing.in(Easing.linear);
+	numbersToTest.forEach((n) => expect(easingIn(n)).toBe(n));
+
+	const easingOut = Easing.out(Easing.linear);
+	numbersToTest.forEach((n) => expect(easingOut(n)).toBeCloseTo(n));
```

Comment:
should be able to use `toBe`, not `toBeCloseTo` I think.

out() is defined as `1 - easing(1 - t)` and linear/easing is just defined as `t => t`.

If we shorten that we get `1 - (1 - t)` --> `1 - 1 + 1` --> `t`.
So `toBe` should work, otherwise we discovered a bug.

Reply:
You're right, if we assume `t=1` then it works. But if we use values like `0.3` or `0.2` it gives some odd results due to floating point calculations. You can reproduce it by just typing `1-1*(1-0.3)` in your JS console. I think a quick workaround would be to swap the test of `0.3` with `0.4` as this does not suffer from floating point precision issues

Reply:
You are right! Thanks for the example, now it makes sense to me!